### PR TITLE
Fix the Doc build script

### PR DESCRIPTION
**Pull request by awholegunch**
Consolidates variables in the vars file.

Some minor fixes. Passes a [Quick Build](http://builds.cask.co/browse/CDAP-DOB115-1).

### CODE REVIEW

File: cdap-docs/build.sh [hunk 1]
```diff
@@ -408,8 +408,16 @@ function build_license_dependency_pdfs() {
 }
 
 function build_standalone() {
```
Comment (anwar6953) [inline cdap-docs/build.sh]: just to clarify - this now requires someone who wants to use this function to have hydrator-plugins cloned, and for it to be at the same level as cdap, right?

Comment (awholegunch) [inline cdap-docs/build.sh]: Correct. Does that make sense? Is it obvious, or do I need to add a comment or to the help?

I guess we could test for the directory and bail if not there.

Comment (anwar6953) [inline cdap-docs/build.sh]: I don't think most people will use this, but it isn't clear to a user that they need hydrator-plugins to build cdap using this command/function.

**Comment by awholegunch** [comment]
@anwar6953 I've changed the script so that the help indicates the requirement for the Hydrator, and where, allows a user to set the location in their own environment instead of using the default, and prints appropriate messages depending on the circumstances.

File: cdap-docs/_common/common-build.sh [hunk 2]
```diff
@@ -66,7 +67,7 @@ TARGET_PATH="${SCRIPT_PATH}/${TARGET}"
 SOURCE_PATH="${SCRIPT_PATH}/${SOURCE}"
 
 if [ "x${2}" == "x" ]; then
-  PROJECT_PATH="${SCRIPT_PATH}/../../"
+  PROJECT_PATH="${SCRIPT_PATH}/../.."
```
Comment (gsps1) [inline cdap-docs/_common/common-build.sh]: why this change? do you know if else wouldn't terminate with `/` ?

Comment (awholegunch) [inline cdap-docs/_common/common-build.sh]: To make it consistent that paths *don't* end in a "/". ``pwd`` doesn't return a path with a slash, so I think this should follow that.

File: cdap-docs/build.sh [hunk 3]
```diff
@@ -49,7 +49,9 @@ function usage() {
   echo "    clean     Clean up any previous build's target directories"
   echo "    javadocs  Build Javadocs"
   echo "    licenses  Clean build of License Dependency PDFs"
-  echo "    sdk       Build CDAP SDK"
+  echo "    sdk       Build CDAP SDK (includes the Hydrator plugins if Hydrator plugins source is at"
+  echo "                              '${HYDRATOR_PLUGINS_PATH}'"
```
Comment (gsps1) [inline cdap-docs/build.sh]: in cdap build readme, we don't specify hydrator, we just say additional artifacts dir, you should follow the same.

Comment (awholegunch) [inline cdap-docs/build.sh]: Except this is specifically for Hydrator. By default, if you add the repo locally, then it will use ``hydrator-plugins``. If you want to pass it a different directory, you can, but this is specifically (in this branch) for building Hydrator. The presence of this directory will trigger the function that builds Hydrator.

File: cdap-docs/build.sh (hunk 1 above)
Comment (awholegunch) [inline cdap-docs/build.sh]: Added code so that it only builds the Hydrator if the directory exists. If not, it prints a message and continues.

**Comment by awholegunch** [comment]
Closing and moving to release 3.4.0.
